test/e2e/components/vl-map.js

```diff
@@ -200,7 +200,7 @@ class VlMap extends VlElement {
    * @see {@link https://openlayers.org/en/latest/apidoc/module-ol_Map-Map.html#getPixelFromCoordinate}
    * @see {@link https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebElement.html#getRect}
    */
-  async _getPixelFromCoordinate(coordinates = [0, 0]) {
+  async getPixelFromCoordinate(coordinates = [0, 0]) {
```

Comment:
Ik zie uitsluitend intern gebruik, dus publiek maken lijkt overbodig?

Reply:
Opgelost door clickOnCoordinates van vl-map te gebruiken in vl-map-draw-point-action